src/App/App.tsx

```diff
@@ -91,208 +92,208 @@ class App extends Component {
                   disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                   exact={true}
                   path="/"
-                  component={Home}
+                  component={WithGATracker(Home)}
```

Comment:
@p-netm true, it's not as DRY as it could be! I tried this quickly and ran into issues: currently the HOC relies on receiving `RouteComponentProps<RouteParams>` as Props to build out the page URL, thus requiring this implementation:
```
<ConnectedPrivateRoute component={WithGATracker(Component)} />
```
I think future iterations / standardization of this code can be agnostic of the react-router and simply build the page URL from `window`.

Reply:
Created a ticket at https://github.com/onaio/js-tools/issues/113 which includes this point as a feature.